### ISSUE TITLE
Passing parameter to push job

### ISSUE THREAD

**Issue by balcantara90** [pantheon-systems/circleci-orb#48]
I'm passing a false clone_content parameter however in circleci I'm getting this error. Not sure what I'm doing wrong

![image](https://user-images.githubusercontent.com/2055259/88769159-6a43ac00-d1bf-11ea-9ed9-82ac31710204.png)

```
  compile_sass_and_push:
    jobs:
      - npmbuild_and_persist
      - pantheon/push:
          clone_content: false
          requires:
            - npmbuild_and_persist
          checkout: false
          pre-steps:
            - checkout
            - run: composer install --no-dev --optimize-autoloader
```

**Comment by stevector** [comment]
Hi @balcantara90, Thanks for trying the orb. The issue is likely the Orb version number. `clone_content` was introduced in `0.5.0`. But the examples in the readme file still used `0.2.0`. I've opened a PR to fix that: https://github.com/pantheon-systems/circleci-orb/pull/49. Can you confirm that this was your issue?

**Comment by balcantara90** [comment]
> Hi @balcantara90, Thanks for trying the orb. The issue is likely the Orb version number. `clone_content` was introduced in `0.5.0`. But the examples in the readme file still used `0.2.0`. I've opened a PR to fix that: #49. Can you confirm that this was your issue?

Rookie mistake, was exactly this. Thanks
